.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,233 @@
+name: build-and-test
+
+on: [push, pull_request]
+
+concurrency:
+  group: build-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  windows-test:
+    name: windows-test
+    runs-on: windows-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Caching dependency
+        uses: actions/cache@v2
+        with:
+          path: |
+            %LocalAppData%\go-build
```

Comment:
Should this match `$env:GOPATH` below?

Reply:
All cache data are stored on `%LocalAppData%\go-build` so I have taken it in cashing section. `$env:GOPATH` will contain all data that are statics between build, Those files are not "cashe" files actually.